test/v3

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+export BUILDPACK_HOME="${BUILDPACK_HOME:-$(pwd)}"
+export CNB_STACK_ID="heroku"
```

Comment:
Do you basically ignore this value? Should this be set by the `Makefile`?

Reply:
it should, and I should honor it instead of STACK in the v3 stuff